tests/test_training_params.py

```
@@ -0,0 +1,365 @@
+
+def _get_train_model_kwarg_names():
+    """Extract keyword-argument names from the model.train_model() call."""
+    call = _get_train_model_call()
+    if call is None:
+        return set()
+    return {kw.arg for kw in call.keywords if kw.arg is not None}
+
+
+def _make_args(**overrides):
+    """Shortcut: create TrainingArguments via the factory with safe defaults."""
+    defaults = dict(output_dir="/tmp/test_gliner_params", use_cpu=True, report_to="none")
```

Ruff (0.15.1)

[error] Probable insecure usage of temporary file or directory: "/tmp/test_gliner_params"

(S108)